src/build/__tests__/build-test.js

```diff
@@ -9,15 +9,30 @@ import {
   GraphQLBoolean,
   GraphQLID,
 } from 'graphql/type';
+import { DirectiveLocation, GraphQLDirective } from 'graphql/type/directives';
+
 import { parse } from 'graphql/language';
 import { expectTypesEqual, expectSchemasEqual } from './comparators';
 import build, { buildTypes } from '../';
 
+const customDirective = new GraphQLDirective({
+  name: 'Dir1',
+  description:
+  'My custom uppercase',
+  locations: [
+    DirectiveLocation.FIELD,
+  ],
+});
 const querySource = `
  type Query {
    ok: Boolean!
  }
 `;
+const querySourceWithDirective = `
```

Comment:
nit: Again, super nit picky (😅) but can you change the name to `queryWithDirectiveSource`?